sdk/lib/collection/hash_map.dart

```diff
@@ -84,8 +84,8 @@ typedef _Hasher<K> = int Function(K object);
 /// To remove an entry with a specific key, use [remove].
 /// ```dart continued
 /// final removeValue = planets.remove(5);
```

Comment:
A smaller change would be to just replace `5` by `6` here and not change subsequent lines.

Reply:
Done